README.md

```diff
@@ -52,21 +55,25 @@ Options:
     -f, --output-format    Requires -o, STDOUT output format: progress, noprogress, machine, stats
     -t, --timeout          Set a timeout in seconds after which stop producing slices
     -r, --router           Set a router: graph_search (default), graph_search_cached
-    -P, --pipeline         pipeline mode: stream (default), dag, wave
-    -g, --graph-search     Set a graph search provider: djikstra (default), astar, boost (not always available) [ignored by -P wave pipeline, which uses astar]
+    -P, --pipeline         pipeline mode: stream (default), wave, edpc, dag (deprecated)
```

Comment:
Just curious, why is `dag` deprecated now? do `edpc` and `wave` fully replace it?

Reply:
I do not know the dag pipeline as well as I do the wave pipeline, but it was my impression that the wave pipeline was an improvement on the dag pipeline in that it constructed the dag in linear time rather than quadratic while effecting the same (or very similar) compilation. What are your thoughts?